src/test/java/com/bytezone/dm3270/TerminalClientTest.java

```diff
@@ -489,6 +447,55 @@ public void shouldGetCorrectFieldsWhenGetFields() throws Exception {
     return screenBuilder.build();
   }
 
+  @Test
```

Comment:
maybe just with the other test is enough for the purpose of the logic being tested?